ambari/install/index-ambari-install.rst

```diff
@@ -0,0 +1,13 @@
+============
+Installation
```

Comment:
Would it make sense to leave installation instruction in the "Installation" chapter and keep the rest (maintencence and so on) here under "Ambari"?